ui/src/app/components/runtime-configuration-panel.rtl.spec.tsx

```diff
@@ -1517,69 +1382,14 @@ describe(RuntimeConfigurationPanel.name, () => {
       expect(customMachineType).not.toEqual(machineType);
       expect(customDiskSize).not.toEqual(persistentDisk.size);
       const { container } = component();
-      expect(getMainCpu(container)).toEqual(
-        findMachineByName(machineType).cpu.toString()
-      );
+
       expect(getMainRam(container)).toEqual(
         findMachineByName(machineType).memory.toString()
       );
       expect(getDetachableDiskValue()).toEqual(persistentDisk.size.toString());
     }
   );
 
-  it(
```

Comment:
equivalent?

Reply:
Added this test back with some conversions.